src/NUnitFramework/testdata/SetUpFixtureData.cs

```diff
@@ -610,3 +610,154 @@ public void Test()
     }
 }
 #endregion
+
+#region SomeFixtureWithExceptions
+[TestFixture]
+public class SomeBaseFixtureExceptionInSetUp
+{
+	private readonly bool exceptionInFixtureSetUp;
+	private readonly bool exceptionInSetUp;
+	public SomeBaseFixtureExceptionInSetUp(): this(false, true)
+	{		
+	}
+	public SomeBaseFixtureExceptionInSetUp(bool exceptionInFixtureSetUp, bool exceptionInSetUp)
```

Comment:
The coding standard is to have one blank line between method blocks to help make it a bit easier to read.